relay/kusama/src/lib.rs

```diff
@@ -1877,8 +1903,8 @@ pub type Block = generic::Block<Header, UncheckedExtrinsic>;
 pub type SignedBlock = generic::SignedBlock<Block>;
 /// `BlockId` type as expected by this runtime.
 pub type BlockId = generic::BlockId<Block>;
-/// The `SignedExtension` to the basic transaction logic.
-pub type SignedExtra = (
+/// The `TransactionExtension` to the basic transaction logic.
+pub type TxExtension = (
```

Comment:
We should add `frame_system::WeightReclaim<Runtime>,` at the end of this pipeline, so the weight unspent in transaction extension is reclaimed from the block weight.

Reply:
Noticed that `WeightReclaim` was introduced on `frame-system` until `stable2503` (`v40.1.0`). Is this correct? Or is it that maybe there is a pending backport for `frame-system` on `stable2412`?